### PR TITLE
feat: introduce multiarch tianon images

### PR DESCRIPTION
**Pull request by pkoutsovasilis**
this PR alternates the used `tianon/true` images to point at the multiarch tag and thus allow native execution on amd64 and arm64 archs

### CODE REVIEW

**Review by jsoriano — APPROVED** [review]
LGTM if CI passes :slightly_smiling_face: Thanks!

**Comment by pkoutsovasilis** [comment]
@jsoriano [this one](https://buildkite.com/elastic/elastic-package/builds/2702#018e7b33-28e8-4455-be34-8f97d7758745) fails but I can't see how terraform deployer even relates to my changes 🥲

**Comment by jsoriano** [comment]
> @jsoriano [this one](https://buildkite.com/elastic/elastic-package/builds/2702#018e7b33-28e8-4455-be34-8f97d7758745) fails but I can't see how terraform deployer even relates to my changes 🥲

Yep, this is unrelated, we are investigating the problem. Feel free to merge in the meantime.

**Comment by pkoutsovasilis** [comment]
I can't merge as the checks need to pass to be able to press the merge button 🥲

**Pull request merged**
